### PR TITLE
Update FAQ

### PR DESCRIPTION
**Pull request by fabienrohrer**
❗️ taget: master

- [x] Improve the General FAQ list to solve more basic issues
- [x] Drop the VisualStudio FAQ item (which can be found easily from google)
- [x] Add a FAQ item about the robot sink
- [x] Support mailto test with Python 3

Preview: https://cyberbotics.com/doc/guide/webots-faq?version=enhancement-faq

### CODE REVIEW

**Comment by fabienrohrer** [comment]
👍

File: docs/guide/general-faq.md [hunk 1]
```diff
@@ -1,23 +1,49 @@
 ## General FAQ
 
-### How Can I Report a Bug in Webots?
+### Is Webots Really Free and Open-Source?
+
+Yes!
+
+Since the [R2019a version](../blog/Webots-2019-a-release.md), Webots is completely free and open-source.
+It remains in active development.
```
Comment (omichel) [inline docs/guide/general-faq.md]:
```suggestion
It remains under active development.
```

File: docs/guide/general-faq.md [hunk 2]
```diff
@@ -1,23 +1,49 @@
...
+Sources are available on [GitHub](https://github.com/cyberbotics/webots) under [the Apache 2 license](https://github.com/cyberbotics/webots/blob/master/LICENSE).
```
Comment (omichel) [inline docs/guide/general-faq.md]:
```suggestion
The source code is available from [GitHub](https://github.com/cyberbotics/webots) and released under the terms of the [Apache 2 license](https://github.com/cyberbotics/webots/blob/master/LICENSE).
```

File: docs/guide/general-faq.md [hunk 3]
```diff
@@ -1,23 +1,49 @@
...
+
+Cyberbotics takes part to many academic and industrial projects, and provides a paid user support, training and consulting for users who need to quickly develop high-quality Webots simulations.
```
Comment (omichel) [inline docs/guide/general-faq.md]:
```suggestion
Cyberbotics collaborates to several academic and industrial projects, and provides a paid user support, training and consulting for users who need to quickly develop high-quality Webots simulations.
```

File: docs/guide/general-faq.md [hunk 4]
```diff
@@ -1,23 +1,49 @@
...
+Please [contact us](mailto:info@cyberbotics.com) for more information about our services.
+
+### Where to Start?
+
+[Download Webots](https://cyberbotics.com/download), [install it](installation-procedure.md) and perform [the official tutorial](tutorials.md).
```
Comment (omichel) [inline docs/guide/general-faq.md]:
```suggestion
[Download Webots](https://cyberbotics.com/download), [install it](installation-procedure.md) and follow the [official tutorials](tutorials.md).
```

File: docs/guide/general-faq.md [hunk 5]
```diff
@@ -1,23 +1,49 @@
...
+
+### Where Can I Find Help?
 
-If you can still start Webots, please report the bug by using Webots menu: `Help / Bug report...`.
+Join [our active community on Discord](https://discordapp.com/invite/nTWbN9m).
+The Cyberbotics team is available (at least) on european buisness hours.
```
Comment (omichel) [inline docs/guide/general-faq.md]:
```suggestion
The Cyberbotics team is available (at least) during European business hours.
```

File: docs/guide/general-faq.md [hunk 6]
```diff
@@ -1,23 +1,49 @@
...
 
-If Webots cannot start any more, please report the bug here: [http://www.cyberbotics.com/bug](http://www.cyberbotics.com/bug).
-Please include a precise description of the problem, the sequence of actions necessary to reproduce the problem.
-Do also attach the world file and the controller programs necessary to reproduce it.
+### Where Can I Ask a Question of General Interest?
+
+On [StackOverflow with the `webots` tag](https://stackoverflow.com/questions/tagged/webots).
+The Cyberbotics team monitor this tag.
```
Comment (omichel) [inline docs/guide/general-faq.md]:
```suggestion
The Cyberbotics team monitors this tag.
```

File: docs/guide/general-faq.md [hunk 7]
```diff
@@ -1,23 +1,49 @@
...
+
+### How Can I Keep Informed?
+
+Follow us on:
+
+- [our Newsletter](https://cyberbotics.com/news/subscribe.php)
```
Comment (omichel) [inline docs/guide/general-faq.md]:
```suggestion
- the [Webots Newsletter](https://cyberbotics.com/news/subscribe.php)
```

File: docs/guide/general-faq.md [hunk 8]
```diff
@@ -1,23 +1,49 @@
...
+### How Can I Report a Bug in Webots?
 
-Before reporting a bug, please make sure that the problem is actually caused by Webots and not by your controller program.
-For example, a crash of the controller process usually indicates a bug in the controller code, not in Webots.
-This situation can be identified by these couple of symptoms:
+Report the bug on [GitHub issues of the Webots main repository](https://github.com/cyberbotics/webots/issues/new/choose).
 
-1. Webots GUI is visible and responsive, but the simulation is blocked (simulation time stopped).
-2. The controller process has vanished from the *Task Manager* (Windows) or is shown as *&lt;defunct&gt;* when using `ps -e` (Linux/Mac).
+Please make sure the issue comes really from Webots and give us simple and precise instructions to reproduce the issue.
```
Comment (omichel) [inline docs/guide/general-faq.md]:
```suggestion
Please make sure the issue comes really from Webots and give us simple and precise instructions to reproduce it.
```

File: docs/guide/general-faq.md [hunk 9]
```diff
@@ -1,23 +1,49 @@
...
 
-Yes.
-However, you will need to create your own project with all the necessary options.
-You will find more detailed instructions on how to do that in [this section](using-visual-studio-with-webots.md).
-To create the import libraries (the "\*.lib" files in Visual Studio) from the "\*.dll" files of the lib directory of Webots, please follow the instructions provided with the documentation of your compiler.
+1. Verify that your system is above the [minimal requirements to run Webots](system-requirements.md).
+2. Update [your GPU drivers](verifying-your-graphics-driver-installation.md). *Webots uses intensively your GPU.*
+3. Update Webots. *Webots is regularly patched to support as much hardware as possible.*
+4. Try the [safe mode](starting-webots.md#safe-mode).
+5. Fill a [crash report](https://github.com/cyberbotics/webots/issues/new/choose).
```
Comment (omichel) [inline docs/guide/general-faq.md]:
```suggestion
5. Fill in a [crash report](https://github.com/cyberbotics/webots/issues/new/choose).
```

**Comment by fabienrohrer** [comment]
@omichel Thank you, I took your comments into account as-is.

**Comment by fabienrohrer** [comment]
👍

**Pull request merged**